packages/experimental/TabList/src/Tab.types.ts

```diff
@@ -0,0 +1,162 @@
+import type * as React from 'react';
+import type { ViewStyle, ColorValue } from 'react-native';
+
+import type { IViewProps } from '@fluentui-react-native/adapters';
+import type { IconProps, IconSourcesType } from '@fluentui-react-native/icon';
+import type { IFocusable, PressableState, PressablePropsExtended } from '@fluentui-react-native/interactive-hooks';
+import type { TextProps } from '@fluentui-react-native/text';
+import type { FontTokens, IBorderTokens } from '@fluentui-react-native/tokens';
+
+export const tabName = 'Tab';
+
+export interface TabTokens extends FontTokens, IBorderTokens {
+  /**
+   * The indicator color.
+   */
+  indicatorColor?: string;
+
+  /**
+   * The opacity of the tabs item.
+   */
+  tabsItemOpacity?: number;
+
+  /**
+   * The indicator marginHorizontal value.
+   */
+  indicatorMarginHorizontal?: number;
+
+  /**
+   * The icon color.
+   */
+  iconColor?: string;
+
+  /**
+   * Source URL or name of the icon to show on the TabsItem.
+   */
+  icon?: IconSourcesType;
+
+  /**
```

Comment:
I don't headerText should be a token. We generally want to only set styling props as tokens